src/utils/tables.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import CustomFooter from '../components/CustomFooter';
+import formatBytes from './formatBytes';
+
+//  Generate MuiTable's columns.
```

Comment:
Please remove commented code

Reply:
Those are comments.  No the code.  I give an example of the param table.  I will remove the comments if you think is good way to go.